designer/server/src/common/templates/partials/navigation.njk

```diff
@@ -3,7 +3,7 @@
 
 {{ appServiceHeader({
   organisationName: "Defra",
-  productName: "Forms designer",
+  productName: "Forms Designer",
```

Comment:
Just highlighting this as product names are typically title case (unlike service names)

See guidance on [**Nunjucks macro options** `productName`](https://design-system.service.gov.uk/components/header/#options-default-1)